app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_out = command.split()[1]
+    file_in = command.split()[2]
```

Comment:
The variable names `file_out` and `file_in` are misleading. According to the task, `file_out` should be the source file and `file_in` should be the destination file. Consider renaming them to `source_file` and `destination_file` for clarity.